.github/workflows/azure-deploy-dev.yml

```diff
@@ -0,0 +1,125 @@
+name: 'HfEYP App Deploy [Azure - DEV]'
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths-ignore:
```

Comment:
The array of files to ignore might want looking at. Some of these files don't exist in this project and should be removed but others might also need to be added in their place.

Reply:
Good catch. Agree on this point.

Reply:
I did look at them and removed the ones below - the ones I kept were either there or could be added but should probably still be ignored.

      - .pa11yci
      - .yardopts
      - uml/*